server/opts.go

```diff
@@ -697,7 +697,20 @@ func (o *Options) processConfigFileLine(k string, v interface{}, errors *[]error
 			}
 			o.TrustedOperators = append(o.TrustedOperators, opc)
 		}
+		// In case "resolver" is defined as well, it takes precedence
+		if o.AccountResolver == nil && len(o.TrustedOperators) == 1 {
+			if accUrl, err := parseURL(o.TrustedOperators[0].AccountServerURL, "account resolver"); err == nil {
+				// accommodate nsc which appends "/accounts" during nsc push
+				suffix := ""
+				if accUrl.Path == "/jwt/v1/" || accUrl.Path == "/jwt/v1" {
+					suffix = "/accounts"
+				}
+				o.AccountResolver, _ = NewURLAccResolver(accUrl.String() + suffix)
+			}
+		}
 	case "resolver", "account_resolver", "accounts_resolver":
+		o.AccountResolver = nil
```

Comment:
I would have added a comment here too, to say that a resolver may have been set when processing "operator", but "resolver" takes precedence, so we clear it here so that if local processing fails, we report the failure.

Reply:
added